src/core_ocean/Registry.xml

```diff
@@ -515,7 +515,7 @@
 					possible_values="any integer"
 		/>
 		<nml_option name="config_cvmix_use_BLD_smoothing" type="logical" default_value=".true." units="NA"
-					description="If true KPP bld is smoothed with a laplacian filter" 
+					description="If true KPP bld is smoothed with a laplacian filter"
```

Comment:
This is where @pwolfram and I differ.  I think this is harmless and might as well be part of this PR.  Presumably this happened automatically in whatever editor @sbrus89 is using and it's more work to go back and undo it than it's worth.

Reply:
@pwolfram, sounds good. I'll change that back.

Reply:
@sbrus89, as @xylar points out, it may not be worth the time so I'm ok either way.  What I think we should do, outside this PR, is to to lint the whole ocean code or just plan to do this incrementally in PRs.  Thoughts on this @mark-petersen and @xylar?

Reply:
Ideally, we would add a linter to GitHub like one can easily do for python code.  Then, we wouldn't have to do all this BS manually.  But, yes, we should do some kind of a linting PR so there isn't so much confusion about who messed up the formatting and when.

Reply:
Maybe this is a job for Stickler CI @xylar (in general outside this PR and for the repo at large)?  Do you know if it is fortran-enabled?  This sounds worthy of a developer discussion.  cc @mark-petersen

Reply:
Wouldn't that be nice.  No, I know for sure that it's not able to handle Fortran.  And I think our linting is pretty non-standard in any case (though we might be able to configure whatever theoretical Fortran linter to do what we want...).